wallet/src/test/java/io/neow3j/wallet/Nep5Test.java

```diff
@@ -0,0 +1,72 @@
+package io.neow3j.wallet;
+
+import io.neow3j.contract.ScriptHash;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.http.HttpService;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.fail;
+
+public class Nep5Test {
+
+    private Neow3j neow3j;
+    private ScriptHash contractScriptHash;
+    private Nep5 nep5;
+
+    @Before
+    public void setUp() {
+        nep5 = Mockito.spy(Nep5.class);
+        this.neow3j = Neow3j.build(new HttpService(""));
+        this.contractScriptHash = new ScriptHash("0x646171fd57e768f473f516d0c931f8d58359b5ea");
+    }
+
+    @Test
+    public void totalSupplyTest() throws IOException {
+        Mockito.doReturn(new BigInteger("66554545")).when(nep5).totalSupply();
+        assertEquals(new BigInteger("66554545"), nep5.totalSupply());
+    }
+
+    @Test
+    public void nameTest() throws IOException {
+        Mockito.doReturn("Nep5 test Name").when(nep5).name();
+        assertEquals("Nep5 test Name", nep5.name());
+    }
+
+    @Test
+    public void symbolTest() throws IOException {
+        Mockito.doReturn("Nep5 test for Symbol").when(nep5).symbol();
+        assertEquals("Nep5 test for Symbol", nep5.symbol());
+    }
+
+    @Test
+    public void decimalsTest() throws IOException {
+        Mockito.doReturn(new BigInteger("66554545")).when(nep5).decimals();
+        assertEquals(new BigInteger("66554545"), nep5.decimals());
+    }
+
+    @Test
```

Comment:
@Priyanshu-soni when you're expecting an exception, can you please use the "expected" attribute in the `@Test` annotation? (instead of doing a try-catch and then fail)
Like here: https://junit.org/junit4/javadoc/latest/org/junit/Test.html